NSS-Day-Cohort-53/tabloid-full-stack-always-polite-iguanas
# Sr view tags

#### Changes Made
1. Added file `TagController.cs` to `Controllers` directory,
2. `Tag.cs` to `Models` directory
3. `TagRepository.cs` and `ITagRepository.cs` to `Repositories` directory
4. Modified file `filename` to include `functionalityname` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sr-view-tags
    ```
4. Open a new Terminal tab (⌘T) and navigate to the server directory.
5. Test app functionality.
    > npm start in `client` directory while API is running
    > Example: When user clicks the `tags` link, then said user should see a list of tags.
6. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
